packages/jellyfish-api-core/src/category/icxorderbook.ts

```diff
@@ -0,0 +1,381 @@
+import { BigNumber, ApiClient } from '../.'
+
+/**
+ * icxrderbook RPCs for DeFi Blockchain
+ */
+export class ICXOrderBook {
+  private readonly client: ApiClient
+
+  constructor (client: ApiClient) {
+    this.client = client
+  }
+
+  /**
+   * Create and submits an ICX order creation transaction.
+   *
+   * @param {Order} order
+   * @param {string} [order.tokenFrom]              Symbol or id of selling token
+   * @param {string} [order.chainFrom]              Symbol or id of selling chain
```

Comment:
```suggestion
   * @param {string} [order.tokenFrom] Symbol or id of selling token
   * @param {string} [order.chainFrom] Symbol or id of selling chain
```

Don't need to format like this, JSDoc can parse and display formatting on its own.

For example:

![image](https://user-images.githubusercontent.com/4266087/121222187-e5362280-c8b8-11eb-8e4f-6b99f70a0442.png)

![image](https://user-images.githubusercontent.com/4266087/121222403-1b73a200-c8b9-11eb-98aa-63b71551bb26.png)